### PR TITLE
Cleanup repo, Add 16.04 path, Fix up documentation

### PR DESCRIPTION
**Pull request by notjames**
Created paths for different distributions. Pre-created ubuntu/{16,18}.04. Added the necessary Nvidia packages for the 16.04 line. The 16.04 piece is untested. The 18.04 side works. Removed the iso from the image.

### CODE REVIEW

File: build/image-creation/packer_images.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Building images with packer
+
+We are building images for MaaS; a general service which can provision operating systems to bare-metal as a service. Thus, we may have a need to many images generally. This article will explain the process we use to build custom images for MaaS using packer. The use of Packer seemed logical for purpose now, but we did notice that using Packer severely increases the time to 'baking' an image.
```
Comment (davidewatson) [inline build/image-creation/packer_images.md]: "Thus, we may have a need to many images generally." -> "Thus, we will need multiple images generally."
"increases the time to 'baking' an image." -> "increases the time to 'bake' an image."

Related: We should use 80 character line limits to make it easier to review PRs.

File: build/image-creation/packer_images.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+The following documentation assumes you're starting from nothing. This repo already has the files for Ubuntu 16.04 and 18.04. However, if you need to extend to a different distribution or version of an exhisting distribution, this documentation will attempt to walk you through that process. If you don't need to create anything new then please skip down to [#Run_packer](Run packer).
```
Comment (davidewatson) [inline build/image-creation/packer_images.md]: This link is broken. You can tell by clicking on the "View file" icon in this review. Maybe it should be :
```
skip down to [Run packer][#run-packer].
```

File: build/image-creation/packer_images.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  $ ssh -i /path/to/<identity key> ubuntu@192.168.2.24
+  ```
+
+  Now follow the import instructions in [](#Import_New_Image_to_MaaS_from_your_workstation)
+
+### Import New Image to MaaS from your workstation
+
+  The following steps are [documented here](https://samsung-cnct.atlassian.net/wiki/spaces/MAAS/pages/323256342/Import+Custom+Image+to+MaaS).
+
+  The following instruction reference a `profile`. This is generally your administrative **username**. It is specifically the name you or someone else provided when creating your maas priv access. For the purposes of this document, the word `profile` will be used for demonstration purposes.
```
Comment (davidewatson) [inline build/image-creation/packer_images.md]: "instruction" -> "instructions"

File: build/image-creation/packer_images.md (hunk 1 above)
Comment (notjames) [reply build/image-creation/packer_images.md]: Fixed; including the 80 chr line limits.

File: build/image-creation/packer_images.md (hunk 2 above)
Comment (notjames) [reply build/image-creation/packer_images.md]: Fixed. Probably didn't create the anchored URL properly for a page that can use markup, but it works.

**Pull request merged**